test/registration/widgets/remote_package_details_test.rb

```diff
@@ -49,4 +49,12 @@
       subject.update(package)
     end
   end
+
+  describe "#clear" do
+    it "empties the widget's content" do
```

Comment:
The method is called `clear` so let's use that word also in the test description
```suggestion
    it "clears the widget's content" do
```